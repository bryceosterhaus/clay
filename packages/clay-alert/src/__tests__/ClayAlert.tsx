/**
 * SPDX-FileCopyrightText: Â© 2019 Liferay, Inc. <https://liferay.com>
 * SPDX-License-Identifier: BSD-3-Clause
 */

describe('ClayAlert', () => {
	it('renders', () => {
		expect(true).toMatchSnapshot();
	});
});
