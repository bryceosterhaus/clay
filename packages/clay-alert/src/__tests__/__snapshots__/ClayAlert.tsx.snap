// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClayAlert renders 1`] = `true`;
